samples/rum-events/vital.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "vital",
+  "date": 1706792746576,
```

Comment:
**question:**
this corresponds to the date where the custom vital has been created?

Reply:
It corresponds to the start of the vital.

Reply:
oh my bad :-( of course!

Reply:
No worries ;)